standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -2537,6 +2544,20 @@ private CreationMetadata convertToCreationMetadata(MCreationMetadata s) {
     return r;
   }
 
+  private SourceTable convertToSourceTable(MMVSource mmvSource) {
+    SourceTable sourceTable = new SourceTable();
+    MTable mTable = mmvSource.getTable();
+    sourceTable.setTableId(mTable.getId());
+    sourceTable.setDbName(mTable.getDatabase().getName());
+    sourceTable.setTableName(mTable.getTableName());
+    String transactionalProp = mTable.getParameters().get(hive_metastoreConstants.TABLE_TRANSACTIONAL_PROPERTIES);
+    sourceTable.setInsertOnly("insert_only".equalsIgnoreCase(transactionalProp));
```

Comment:
this `equalIgnoreCase` is a bit out of scope here: maybe there is a method which could be used for this?

Reply:
There is `org.apache.hadoop.hive.ql.io.AcidUtils.isInsertOnlyTable` but that method is not accessible here because `ql` is not referenced from `metastore-server`.
Should it be moved to the `common` module?